connectors/connector-greenhouse/def.ts

```diff
@@ -0,0 +1,24 @@
+import type {ConnectorDef, ConnectorSchemas} from '@openint/cdk'
+import {connHelpers} from '@openint/cdk'
+import {z} from '@openint/util'
+
+export const greenhouseSchema = {
+  name: z.literal('greenhouse'),
+  connectorConfig: z.object({apiKey: z.string()}),
```

Comment:
This should be in `resourceSettings` rather than `connectorConfig`. ConnectorConfig would be for things that are on owned by our direct users (e.g. oauth client id & secrets) rather than end users (oauth access / refresh token)